microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/integration/FailSafeCDISystemPropertyHealthTest.java

```diff
@@ -0,0 +1,87 @@
+ */
+@RunWith(Arquillian.class)
+@RunAsClient
+@ServerSetup({ MicroProfileFTSetupTask.class, FailSafeCDISystemPropertyHealthTest.SetupTask.class })
+public class FailSafeCDISystemPropertyHealthTest extends FailSafeCDIHealthBaseTest {
+
+    private static final PathAddress LIVE_SYSTEM_PROPERTY_ADDRESS = PathAddress.pathAddress()
+            .append(SYSTEM_PROPERTY, FailSafeDummyService.LIVE_CONFIG_PROPERTY);
+
+    private static final PathAddress READY_SYSTEM_PROPERTY_ADDRESS = PathAddress.pathAddress()
+            .append(SYSTEM_PROPERTY, FailSafeDummyService.READY_CONFIG_PROPERTY);
+
+    private static final PathAddress IN_MAINENANCE_SYSTEM_PROPERTY_ADDRESS = PathAddress.pathAddress()
+            .append(SYSTEM_PROPERTY, FailSafeDummyService.IN_MAINTENANCE_CONFIG_PROPERTY);
+
+    private static final PathAddress READY_IN_MAINENANCE_SYSTEM_PROPERTY_ADDRESS = PathAddress.pathAddress()
+            .append(SYSTEM_PROPERTY, FailSafeDummyService.READY_IN_MAINTENANCE_CONFIG_PROPERTY);
+
+    @Deployment(testable = false)
+    public static WebArchive createDeployment() {
+        WebArchive webArchive = ShrinkWrap
+                .create(WebArchive.class, FailSafeCDISystemPropertyHealthTest.class.getSimpleName() + ".war")
+                .addClasses(FailSafeDummyService.class, CDIBasedLivenessHealthCheck.class, CDIBasedReadinessHealthCheck.class)
+                .addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml");
+        return webArchive;
+    }
+
+    @Override
+    void setConfigProperties(boolean live, boolean ready, boolean inMaintanance, boolean readyInMainenance) throws Exception {
+        OnlineManagementClient client = ManagementClientProvider.onlineStandalone();
+        client.execute(String.format("/system-property=%s:write-attribute(name=value, value=%s)",
+                FailSafeDummyService.LIVE_CONFIG_PROPERTY, live));
+        client.execute(String.format("/system-property=%s:write-attribute(name=value, value=%s)",
+                FailSafeDummyService.READY_CONFIG_PROPERTY, ready));
+        client.execute(String.format("/system-property=%s:write-attribute(name=value, value=%s)",
+                FailSafeDummyService.IN_MAINTENANCE_CONFIG_PROPERTY, inMaintanance));
+        client.execute(String.format("/system-property=%s:write-attribute(name=value, value=%s)",
+                FailSafeDummyService.READY_IN_MAINTENANCE_CONFIG_PROPERTY, readyInMainenance));
+        new Administration(ManagementClientProvider.onlineStandalone()).reload();
+    }
+
+    /**
+     * Add system properties for MP Config
+     */
+    public static class SetupTask implements ServerSetupTask {
+
+        @Override
+        public void setup(ManagementClient managementClient, String s) throws Exception {
+            OnlineManagementClient client = ManagementClientProvider.onlineStandalone();
+            client.execute(String.format("/system-property=%s:add", FailSafeDummyService.LIVE_CONFIG_PROPERTY));
```

Comment:
Is there a reason why are you using raw CLI instead of using `org.wildfly.extras.creaper.core.online.operations.Address` and `org.wildfly.extras.creaper.core.online.operations.Operations`? Do you check for success on these operations?

Reply:
IMO cli operation is more readable and looks simpler to to someone not familiar with creaper. However I can change it if you want.
Fixing checking for success.